doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -123,60 +119,50 @@ emitters:
 - `clear-output-folder` indicate if you want to clear up the output folder.
 
 
-#### Create a sdk project folder
+#### Create sdk starter package
```

Comment:
```suggestion
#### Create the library project
```

Reply:
Hello @jsquire  A .Net sdk contains many content besides the generated source code, after we automatically generate the sdk package with necessary directories, customer/service team need to follow the following guide to manually add tests/samples/changelog and so on to meet the release requirement. So we decided to call this first initialized sdk package `starter package`.

Reply:
"Package" is an overloaded term.    This does not create a NuGet package, which is my concern about potential confusion.    I'm not pushing for "library project" - but we need a term other than "package" here.